docs/sdk/api/apiRequestBuilder.md

```diff
@@ -14,38 +14,47 @@ npm install --save @commercetools/api-request-builder
 <script>// global: CommercetoolsApiRequestBuilder</script>
 ```
 
-## `createRequestBuilder(customServices)`
+## `createRequestBuilder(options)`
 
 Creates a *request builder* that allows to declaratively build a HTTP API request URI for the commercetools platform.
 
 #### Arguments
 
-1. `customServices` *(Object)*: A map of custom services that are not provided by default. This might be useful to build a request for a different API with similar query parameters.
+The `options` argument must be an object with a `projectKey` property, and optionally a `customServices` property
+```js
+    options -> { projectKey, customServices }
+```
+
+1. `projectKey` *(String)*: A required staring specifying the project key to use for the request, with the format. Even though this is required, the project key can be omitted from the URI by passing `{withProjectKey: false}` to `.build()`
```

Comment:
oops... typo. 🙈 
But is that the only thing? So I could fix it all in one commit